src/PIL/Image.py

```diff
@@ -122,20 +122,6 @@ class DecompressionBombError(Exception):
     raise
 
 
-def isImageType(t: Any) -> TypeGuard[Image]:
-    """
-    Checks if an object is an image object.
-
-    .. warning::
-
-       This function is for internal use only.
```

Comment:
This [11-year-old docstring](https://github.com/python-pillow/Pillow/commit/15eac684c4ef1ce38a2c95d259c60b3d46c803d3#diff-68bb791db6b090ce3dc87eb429f6b033085c0a4acd8e36894880e35db9e02cdbR99) may well have been warning this is for internal use only but the function is [24 years old, since PIL 1.1.1,](https://github.com/hugovk/PIL/blame/645e19c47556b9086c0d0fdccc1db1b9e6f79f7f/PIL/Image.py#L98) and is [used in the wild](https://grep.app/search?current=2&q=isImageType&case=true&filter[lang][0]=Python).

To be on the safe side, shall we put this through a deprecation cycle?

Reply:
Sure. I've pushed updates.